assets/src/frontend/components/Home.vue

```diff
@@ -923,7 +923,26 @@ export default {
                 this.productLoading = false;
             }
         },
+        appendProducts( products ) {
+            products.forEach( product => {
+                if ( "variable" === product.type && this.isAllVariationsDisabled( product ) ) {
+                    return;
+                }
+
+                this.products = this.products.concat( product );
+            });
+        },
+        isAllVariationsDisabled( product ) {
+            let isDisabled = true;
 
+            product.attributes.forEach( attribute => {
+                if ( true === attribute.variation ) {
```

Comment:
Kindly make sure the value of the `attribute.variation` here is `boolean true`. If it may returned as `string 'true'` or `int 1` or `string '1'`, there could be inconsistency.

Reply:
Bhai, the `attribute.variation` is coming from `WooCommerce` products API. And it always returns `boolean`.